querydsl-sql-codegen/src/main/java/com/querydsl/sql/codegen/ExtendedBeanSerializer.java

```diff
@@ -28,6 +28,8 @@
 /**
  * {@code ExtendedBeanSerializer} outputs primary key based {@code equals}, {@code hashCode} and
  * {@code toString} implementations
+ * 
+ * Requires column annotation generation to be enabled
```

Comment:
I'll wait for @timowest or @johnktims to chime in for the text.
But there is a trailing space at line 31.